### DISCUSSION TITLE
Leave Min SoC (Off-Grid) at previous value

### DISCUSSION THREAD

**Discussion by IB-UK** [nathanmarlor/foxess_em#144] [Ideas]
AIUI the Min SoC is adjusted as a mechanism to force the battery to charge.  Is there a technical reason why BOTH on and off grid values are set to 99%?  In this example if there is a grid outage overnight my EPS would fail as the battery would not want to drop below the Min SoC of 99%.

Could it be an option to leave the Min SoC at the previous value (10% in my case) it shouldn't alter the logic of forcing a charge but it would mean the inverter would allow the battery to discharge in the event of a power outage during a force charge time.

**Comment by nathanmarlor** [comment]
@IB-UK this should now be resolved in the latest 1.6.0 beta :)

**Reply by IB-UK** [reply]
@nathanmarlor Superb work, thanks for that.
